### PR TITLE
README edits

### PR DESCRIPTION
**Pull request by sjschmidt**
Tried to adapt the README.md to be more legible and useful. I was unsure how to fully describe the functions so it would be useful if others would adapt.

### CODE REVIEW

File: Schmidt/template/README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+
+factorial is a function that returns something important.
```
Comment (ojhall94) [inline Schmidt/template/README.md]: What is a factorial? Could you maybe include the mathematical formulate, eg:

factorial(10) returns 10!

File: Schmidt/template/README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
     >>> from simple_functions import factorial
     >>> factorial(10)
     9
```
Comment (ojhall94) [inline Schmidt/template/README.md]: 10! is not 9, this should be 3,628,800 instead.

File: Schmidt/template/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-and this other part does something.  I forget why that I did it:
+The function fibonnacci returns a fibonacci sequence of all numbers that are less than the given numbers. 
 
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
+
```
Comment (ojhall94) [inline Schmidt/template/README.md:14]: It's important we know who's hard work went into these changes, you should add yourself as author at the bottom of the readme.

**Review by ojhall94 — CHANGES_REQUESTED** [review]
Left a few minor comments!

File: Schmidt/template/README.md (hunk 2 above)
Comment (sjschmidt) [reply Schmidt/template/README.md]: done!

File: Schmidt/template/README.md (hunk 3 above)
Comment (sjschmidt) [reply Schmidt/template/README.md:14]: Done!

File: Schmidt/template/README.md (hunk 1 above)
Comment (sjschmidt) [reply Schmidt/template/README.md]: Cool.

**Review by ojhall94 — APPROVED** [review]
These changes are great! Good job. Gold star.

**Comment by bsipocz** [comment]
@sjschmidt @ojhall94 - Thanks for opening the pull request and for the review(s)!  Everything looks good, so we'll go ahead and merge this. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. 

This is a nicely done full process of PR, review, follow-up, approve.

File: Schmidt/template/README.md [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
+    3628800
 
-and this other part does something.  I forget why that I did it:
+The function fibonnacci returns a fibonacci sequence of all numbers that are less than the given numbers. 
 
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
+Readme updated 27.08.2019 by Sarah Jane Schmidt
```
Comment (eteq) [inline Schmidt/template/README.md:13]: @sjschmidt - one minor comment here: in general on git it's not a good idea to use "dated" information like this - the history of updates of a file are always available via `git log README.md`, so this is not necessary, and text like this has a tendancy to get out-of-date (i.e., if someone *other* than you edited this readme, they might not think to add to this file).  So it's best to just leave this sort of information off from anything tracked in git unless there's a specific known need for it.

**Pull request merged**